addons/mail/views/mail_template_views.xml

```diff
@@ -5,44 +5,36 @@
             <field name="name">email.template.form</field>
             <field name="model">mail.template</field>
             <field name="arch" type="xml">
-                <form string="Templates">
+                <form string="Templates" js_class="mail_template_form">
                     <header>
+                        <field name="ref_ir_act_window" invisible="1"/>
                         <field name="template_fs" invisible="1"/>
+                        <field name="is_template_editor" invisible="1"/>
+                        <button string="Preview"
+                                name="%(mail_template_preview_action)d" type="action"
+                                class="btn-primary" target="new"/>
                         <button string="Reset Template"
                                 name="%(mail_template_reset_action)d" type="action"
                                 groups="mail.group_mail_template_editor"
                                 invisible="not template_fs"/>
+                        <!-- onclick added by js_class for confirmation modal -->
+                        <span class="btn btn-secondary btn-group delete-mail-template"
```

Comment:
Side note: "delete-template-record" seems weird, not sure what are current conventions in JS but would go for a more "o_mail_template_delete" class

Reply:
It won't be needed anymore if we don't delete records from the wizard so keeping it like that for now